### ISSUE TITLE
question on the file model_manager.py

### ISSUE THREAD

**Issue by appleleaves** [antspy/quantized_distillation#9]
In the file `model_manager.py` 
`        if continue_training_from > 0:
            #if continue_training_from == 0 then the model has never been trained before
            #(it has just been added as a new model) so we load only when continue_training_from > 0
            print('Loading weights from train run number {}'.format(continue_training_from))
            state_dict = torch.load(self.saved_models[model_name][continue_training_from].path_saved_model)
            try:
                model.load_state_dict(state_dict)
            except KeyError:
                #this means that the weight were saved with a DataParallel but the current one is not
                #or vice-versa.
                if isinstance(model, torch.nn.parallel.DataParallel):
                    state_dict = mhf.convert_state_dict_to_data_parallel(state_dict)
                else:
                    state_dict = mhf.convert_state_dict_from_data_parallel(state_dict)
                model.load_state_dict(state_dict)`

you load the `state_dict` only but fail to load the learning rate and other parameters, it is not continue_training but train with a better initialization. Am I right? Is there any mistakes?
In this way, the continue training is wired. Few paper(including yours) explain that we should train a model again in this way. 
Can you explain your code to me?

**Comment by antspy** [comment]
Hi, 

you are correct, I do not save the optimizer (which would include learning rate and other parameters). This is a limitation of the code, you are welcome to contribute and add the necessary code to the class (note though that saving the optimizer would ~double the disk space occupied)
